reference_simulations/compare_with_last.py

```diff
@@ -0,0 +1,37 @@
+#!/bin/python
+
+# Compare the visibilities in a given set of simulation outputs to the corresponding
+# values in a reference set.
+
+import numpy as np
+import sys
+import glob
+import warnings
+import os
+
+import h5py
+
+# Loading data using h5py directly instead of reading the full uvh5 files.
+#   (The 1.2 reference sims recalculate all lsts when reloaded, which takes a long time.)
+
```

Comment:
This recalculation can be turned off if the `lst_array` is already in the file by setting `run_check_acceptability=False`. But if they're not in the file there's no way to turn it off.

Reply:
Thanks for the tip! With that in place, I can use pyuvdata instead of h5py, so I've switched to that.